src/v3/src/transformer/uischema/buildAriaDescribedByValues.ts

```diff
@@ -0,0 +1,41 @@
+/*
+ * Copyright (c) 2022-present, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import { TransformStepFn, UISchemaElement } from '../../types';
+import { isInteractiveType } from '../../util';
+import { traverseLayout } from '../util';
+
+export const updateElementsWithAriaDescribedByValues: TransformStepFn = (formbag) => {
+  const descriptionElements: UISchemaElement[] = [];
+  traverseLayout({
+    layout: formbag.uischema,
+    predicate: (el) => el.type === 'Title' || el.contentType === 'subtitle',
+    callback: (el) => {
+      descriptionElements.push(el);
+    },
+  });
+
+  traverseLayout({
+    layout: formbag.uischema,
+    predicate: (el) => isInteractiveType(el.type),
+    callback: (el) => {
+      const descriptiveElementIds = descriptionElements
+        .filter(
+          (descrEle: UISchemaElement) => typeof descrEle.viewIndex === 'undefined'
+            || descrEle.viewIndex === el.viewIndex,
```

Comment:
I think I understand why we have `viewIndex` but can you give a short explanation?

Reply:
I added the explanation as a ts doc to the variable in the interface. Is this ok or should I move it directly here? I chose the ts doc incase someone uses it in the future, they can quickly see the purpose of the variable. 
![image](https://user-images.githubusercontent.com/97472729/206274490-74f1460b-da72-4884-aaa2-5e5e7482615f.png)